src/commands/txCommands/util/pushTxAllTranslations.js

```diff
@@ -7,7 +7,10 @@ const pushTxAllTranslations = () => {
   txlib.updateAllTranslationsResource$
     .subscribe(
         null,
-        (error) => console.error(`encountered error during upload: ${error}`),
+        (error) => {
+            console.error(`encountered error during upload: ${error}`);
+            process.exit(1);
```

Comment:
💯  

On a separate note, can't believe I didn't think of process.exit() when we were pairing and googling node solutions 🤦‍♀️  ...